084_2_kaimMatricaPapildykBriauna.cpp

```diff
@@ -0,0 +1,6 @@
+void papildykBriauna (int u, int v) { // Kadangi grafo kaimynystės matricą saugojomės

```

Comment:
Siūlau komentarą rašyti taip, multiline gal šiek tiek gražiau žiūrėsis nei šone kabantis:

```cpp
void papildykBriauna(int u, int v) {
    /*
       Kadangi grafo kaimynystės matricą saugojomės
       globaliai (žr. ankstesnį kodą), nereikia
       paties grafo paduoti funkcijos parametruose.
    */
    ...
}
```

Jei keisi, tai atnaujink ir kitas vietas.

Reply:
Gerai, pakeisiu. Ar tuomet ir, pavyzdžiui, šitokią vietą:
```
int spalva[MAXN];      // spalva[i] yra 0, jei i-tosios viršūnės spalva yra balta,
                       //               1 - jei pilka,
                       //               2 - jei juoda
                       // pradinės reikšmės - 0
int pirmine[MAXN];     // prieš kviečiant DFS reiktų nustatyti pradines reikšmes į -1
```
reiktų keisti į kažką tokio?
```
int spalva[MAXN];      
/*
   spalva[i] yra 0, jei i-tosios viršūnės spalva yra balta,
                 1 - jei pilka,
                 2 - jei juoda
   pradinės reikšmės - 0
*/

int pirmine[MAXN];     // prieš kviečiant DFS reiktų nustatyti pradines reikšmes į -1
```

Reply:
Šitą konkrečią vietą manau gali palikt.